docs/reference/workflows/training.md

```diff
@@ -106,4 +106,25 @@ In `stage_1`, the model learns to classify from extracted features. In `stage_2`
 
 If you are training images that are similar to the ones in the COCO dataset, it is best to train more epochs in `stage_1`, otherwise it's best to train more epochs in `stage_2` where the model is learning to better extract features.
 
-If you have ~1000 images, then you don't have to change any parameters.
\ No newline at end of file
+If you have ~1000 images, then you don't have to change any parameters.
+
+## Fine-tuning and checkpoints
+
+Fine-tuning is a common technique for transfer learning. The target model copies all model designs with their parameters from the source model except the output layer, and fine-tunes these parameters based on the target dataset.
+
+
+To do this:
```

Comment:
Reword to:
To fine-tune a model or continue from a checkpoint from previous training Workflow execution: